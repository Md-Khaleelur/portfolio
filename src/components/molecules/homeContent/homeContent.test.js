import React from 'react';
import { create } from "react-test-renderer";
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import HomeContent from './homeContent';

describe("HomeContent component",() =>{
    test("Matches the snapshot",() => {
        const homeContent = create(<HomeContent/>);
        expect(homeContent.toJSON()).toMatchSnapshot();

    })
    test("HomeContent is rendered",() => {
        const { getByTestId, getByText } = render(<HomeContent/>);
        expect(getByTestId("homeContent")).toBeInTheDocument();
        
    });
   

});