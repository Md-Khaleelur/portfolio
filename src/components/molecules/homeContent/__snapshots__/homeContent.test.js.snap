// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContent component Matches the snapshot 1`] = `
<div
  data-testid="homeContent"
>
  <div
    className="index"
  >
    <div
      className="wrapper"
    >
      <div
        className="img-info"
      >
        <div
          data-testid="title"
        >
          <h2>
            MOHAMMED KHALEELUR REHMAN
          </h2>
        </div>
        <div
          data-testid="para"
        >
          <p>
            I am Mohammed Khaleelur Rehman, an undergraduate Computer Science Engineering student at Muffakham Jah College of Engineering and Technology, Hyderabad. I am learning to become a full stack web developer.
          </p>
        </div>
        <div
          data-testid="para"
        >
          <p>
            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
          </p>
        </div>
      </div>
      <div
        data-testid="imageContent"
      >
        <img
          alt="img-miss"
          src="me2.png"
        />
      </div>
    </div>
  </div>
</div>
`;
